src/qibolab/oneQ_emulator.py

```diff
@@ -0,0 +1,56 @@
+from qibolab.emulator import create_runcard_emulator
+from qibolab.instruments.simulator.models import general_no_coupler_model
+from qibolab.instruments.simulator.pulse_simulator import (
+    PulseSimulator,
+    get_default_simulation_config,
+)
+
+# device parameters: frequency in Hz, time in s
+device_name = "ibmfakebelemQ0"
+sampling_rate = 4500000000.0
+readout_error = [0.01, 0.02]
+lo_freq = 5090167234.445013
+anharmonicity = -336123005.1821652
+drive_freq = 5090167234.445013
+rabi_freq = 125457538.19061986
+T1 = 8.857848970762537e-05
+T2 = 0.00010679794866226273
+nlevel = 3
+
+# simulation parameters
+sim_sampling_boost = 10
+simulate_dissipation = True
+instant_measurement = True
+
+
+def create_oneQ_emulator(runcard_folder: str):
+    """Create a one qubit emulator platform."""
+
+    model_params_dict = {
+        "device_name": device_name,
+        "topology": [],
+        "nqubits": 1,
+        "ncouplers": 0,
+        "qubits_list": ["0"],
+        "couplers_list": [],
+        "sampling_rate": sampling_rate,
+        "readout_error": {0: readout_error},
+        "drive_freq": {"0": drive_freq},
+        "T1": {"0": T1},
+        "T2": {"0": T2},
+        "lo_freq": {"0": lo_freq},
+        "rabi_freq": {"0": rabi_freq},
+        "anharmonicity": {"0": anharmonicity},
+        "coupling_strength": {},
+    }
+
+    model_config = general_no_coupler_model.generate_model_config(
+        model_params_dict, nlevels_q=[nlevel]
+    )
+    simulation_config = get_default_simulation_config(sim_sampling_boost)
+    simulation_config.update({"simulate_dissipation": simulate_dissipation})
+    simulation_config.update({"instant_measurement": instant_measurement})
```

Comment:
For the sake of clarity and reproducibility, I'd tend to avoid having functions generating the defaults, and just dump the resulting `model_config` and `simulation_config` in the `model.json` (and possibly `simulation.json`, if needed).
Instead, if you need helpers to ease serialization and deserialization, for them Qibolab is the optimal place.

Reply:
So the `default_simulation_config` is the recommended settings that guide an unfamiliar user who might be interested to play around with some of these settings. If not for having such functions, how else would you suggest that I indicate or store these recommended options and values?

Reply:
> If not for having such functions, how else would you suggest that I indicate or store these recommended options and values?

Documentation :)

Reply:
Fixed in 2a7c713. simulation_config is now constructed in `platform.py` in the emulator platform folder